network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html

```diff
@@ -10,15 +10,16 @@
 {% block subcontent %}
 <div class="tw-container">
   <div class="tw-row">
-    <div class="
+    <div class="{% if page.aside_page_content %}col-lg-8{% endif %}"> <!-- conditional side wrapper-->
+      <div class="
       {% block content_wrapper_class %}tw-py-5 medium:tw-py-7{% endblock %}
       {% if page.specific.narrowed_page_content %}
         tw-w-8/12 tw-mx-auto tw-px-4
       {% else %}
         tw-w-full tw-px-4
       {% endif %}"
```

Comment:
Have you considered merging these two wrapper elements? It seems like the previously existing one already does the same thing for the narrow page content variation. Could we achieve the same as right now with an addition `ifelse` tag?

Reply:
Thanks, I looks like narrow content does exactly what we want here so I've added `aside` in with that `if` check 7eef84c36986ca3bc0ec51aa616ee7697e1a0723